script.js

```diff
@@ -118,29 +118,57 @@ function dispMessages() {
         'orderBy': 'startTime',
         'showDeleted': false,
         'singleEvents': true,
-        'timeMin': tMin.toISOString(), // mky: Date has to be an object with new
+        'timeMin': tMin.toISOString(), // Date has to be an object with new
         'timeMax': tMax.toISOString(),
     }).then(function (response) {
-        var events = response.result.items;  // MKy: The array of Event-objects
-        document.getElementById("messages").innerHTML = "";
-
-        events.reverse();  // cause orderBy endtime doesn't exist.
+        var noImage = 1;
+        var events = response.result.items;  // The array of Event-objects
+        events.reverse();  // because orderBy endtime doesn't exist.
 
+        // initialisation of messages and aside
+        document.getElementById("messages").innerHTML = "";
+        if ((asd = document.getElementById("myAside"))) {  //not good practice the "=", but compact
+            asd.parentNode.removeChild(asd);
+        }
+        // screening and handling events
         if (events.length > 0) {
             for (i = 0; i < events.length; i++) {
-                var para = document.createElement("P");
-                para.innerHTML = events[i].summary;
-                document.getElementById("messages").appendChild(para);
+                if (!events[i].attachments) {
+                    var para = document.createElement("P");
+                    para.innerHTML = events[i].summary;
+                    document.getElementById("messages").appendChild(para);
+                }
+                else if (noImage) {
+                    var attachType = events[i].attachments[0].mimeType;
+                    if (attachType.indexOf("image") !== -1) {
```

Comment:
Je crois il y a une manière plus propre de faire ça. Tu veux juste t'assurer que "image" fait partie de attachType, non ?
Bon après, si ça marche, laisse comme ça. "Don't fix it if it ain't broken !"